modules/phase_field/include/ics/SmoothSuperellipsoidBaseIC.h

```diff
@@ -37,6 +37,7 @@ class SmoothSuperellipsoidBaseIC : public InitialCondition
 
 protected:
   virtual Real computeSuperellipsoidValue(const Point & p, const Point & center, const Real & a, const Real & b, const Real & c, const Real & n);
+  virtual Real computeSuperellipsoidInverseValue(const Point & p, const Point & center, const Real & a, const Real & b, const Real & c, const Real & n);
   RealGradient computeSuperellipsoidGradient(const Point & p, const Point & center, const Real & a, const Real & b, const Real & c, const Real & n);
```

Comment:
I prefer if all const refs to Reals were changed to pass by value, i.e.

```
virtual Real computeSuperellipsoidInverseValue(const Point & p, const Point & center, const Real & a, const Real & b, const Real & c, const Real & n);
```

becomes

```
virtual Real computeSuperellipsoidInverseValue(const Point & p, const Point & center, Real a, Real b, Real c, Real n);
```

or we might want to think about passing the axis as a `RealVectorValue` to reduce the number of arguments